### PR TITLE
Bumps & housekeeping

### PR DESCRIPTION
**Pull request by asoltysik**
Preelminary PR

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -10,20 +10,24 @@
...
-    crossScalaVersions := Seq("2.10.7", "2.11.12", "2.12.3"),
-    scalacOptions      := Seq("-deprecation", "-encoding", "utf8"),
-    javacOptions       ++= Seq("-source", "1.8", "-target", "1.8")
+    organization := "com.snowplowanalytics",
+    version := "0.5.0",
+    description := "Scala tracker for Snowplow",
+    name := "snowplow-scala-tracker",
+    scalaVersion := "2.12.6",
+    crossScalaVersions := Seq("2.11.12", "2.12.6"),
+    scalacOptions := Seq("-deprecation", "-encoding", "utf8"),
```
Comment (chuwy) [inline build.sbt]: Can we add some options for `scalac` safety (`infer-any`, `dead-code` etc)

**Review by chuwy — APPROVED** [review]
👍

**Comment by asoltysik** [comment]
@chuwy comment adressed

**Review by BenFradet — APPROVED** [review]
:+1:

**Pull request merged**